frontend/src/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -433,6 +434,12 @@ export class SummaryPanelNodeComponent extends React.Component<SummaryPanelNodeC
             </span>
           </div>
         )}
+        {!isGateway && isAutogenerated && (
+          <div>
+            <KialiIcon.Gateway />
+            <span style={{ paddingLeft: '4px' }}>K8s Gateway API</span>
+          </div>
+        )}
```

Comment:
I think I would prefer it if this info is appended to the right.  So, similar to what is done for `Is Gateway (K8s GW API)` we could have `Has Request Routing (K8s GW API)`.    This would be more consistent and improve locality of the information.

Reply:
This was my initial implementation, but I thought, that from UX perspective too long sentence could be not nice, but I am fine to change it.

Reply:
Then we need to have it on all, even not K8s GW supported routing types.